specification.json

```diff
@@ -675,7 +675,7 @@
         {
             "id": "Requirement 5.2.3",
             "machine_id": "requirement_5_2_3",
-            "content": "The `event details` MUST contain the `client name` associated with the event.",
+            "content": "The `event details` MUST contain the `provider name` associated with the event.",
```

Comment:
This makes sense. Having this in handler callback helps with correlating events with the provider.

Reply:
resolving as done